operations/finance.md

```diff
@@ -743,6 +759,156 @@ Removes selected bills. Bill must be empty, otherwise it's not possible to delet
+| `TaxIdentifier` | string | optional | Tax identifier of account to be printed on a bill. |
+| `Notes` | string | optional | Notes to be attached to bill. |
+| `Address` | [Address parameters](customers.md#address-parameters) | optional | Address of the account to be displayed on bill. Overrides the default one taken from account profile. |
+
+### Response
+
+```javascript
+{
+    "Bills": [
+        {
+            "Id": "44eba542-193e-47c7-8077-abd7008eb206",
+            "CustomerId": "7eaf9da6-7229-454a-8cb0-abd700804bd2",
+            "CompanyId": null,
+            "CounterId": "84b25778-c1dd-48dc-8c00-ab3a00b6df14",
+            "State": "Closed",
+            "Type": "Receipt",
+            "Number": "84",
+            "VariableSymbol": null,
+            "CreatedUtc": "2020-06-11T08:39:32Z",
+            "IssuedUtc": "2020-06-25T08:49:38Z",
+            "TaxedUtc": "2020-06-25",
+            "PaidUtc": null,
+            "DueUtc": null,
+            "Notes": null,
+            "Options": [
+                "DisplayCustomer",
+                "DisplayTaxation"
+            ],
+            "Revenue": [],
+            "Payments": []
+        }
+    ]
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `Bills` | array of [Bill](#bill) | required | The closed bill. |
+
+## Get bill PDF
+
+Prints PDF version of closed bill. In case it's not possible to return PDF immediately returns unique event identifier which must be used in retried requests. 
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/bills/getPdf`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "BillId": "44eba542-193e-47c7-8077-abd7008eb206",
+    "BillPrintEventId": null
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `BillId` | string | required | Unique identifier of the [Bill](finance.md#bill) to be printed. |
+| `BillPrintEventId` | string | optional | Unique identifier of the [Bill print event](#bill-print-event) returned by previous operation call. |
+
+### Response
+
+```javascript
+{
+    "BillId": "44eba542-193e-47c7-8077-abd7008eb206",
+    "Result": {
+        "Discriminator": "BillPdfFile",
+        "Value": {
+            "Base64Data": "JVBER....."
+        }
+    }
+}
+```
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `BillId` | string | required | The unique identifier of printed [Bill](finance.md#bill). |
+| `Result` | object [Bill PDF result](#bill-pdf-result) | required | The result of operation. |
+
+#### Bill PDF result
```

Comment:
Don't we call result here  data in other places when there is a discriminator ?

Reply:
I think it's OK as it is. In this usage it's either data (pdf file) or just some ID in case it's not possible to print do the result seem more general to me.

Reply:
It really depends on what the coproduct represents. On the other places it represents some additional data of an entity so data suffix makes sense. Here it represents the result so the current naming is good.